node/network/bridge/src/lib.rs

```diff
@@ -556,6 +563,15 @@ where
 						network_service = ns;
 						authority_discovery_service = ads;
 					}
+					NetworkBridgeMessage::GetAuthorityDiscoveryService(tx) => {
+						if let Err(err) = tx.send(Box::new(authority_discovery_service.clone())) {
```

Comment:
Can we pass it at the configuration instead? Sending `Box<Arc<AuthorityDiscoveryService>>` on a request seems weird.

Reply:
Yeah, that makes a lot of sense!